miden/tests/integration/main.rs

```diff
@@ -104,24 +104,29 @@ macro_rules! build_test_by_mode {
         $crate::helpers::Test::new($source, $in_debug_mode)
     }};
     ($in_debug_mode:expr, $source:expr, $stack_inputs:expr) => {{
-        let inputs = $crate::helpers::ProgramInputs::new($stack_inputs, &[], vec![]).unwrap();
+        let stack: Vec<u64> = $stack_inputs.to_vec();
+        let stack = $crate::helpers::StackInputs::try_from_values(stack).unwrap();
+        let inputs = $crate::helpers::ProgramInputs::new(&[], vec![]).unwrap();
```

Comment:
nit: would prefer to rename the variables (here and below) as:
- `stack` -> `stack_inputs`.
- `inputs` -> `advice_inputs`.

Reply:
Fixed!